Cargo.toml

```
@@ -0,0 +1,41 @@
+[package]
+name = "satomic-swap"
+version = "0.1.0"
+edition = "2021"
+
+[lib]
+crate-type = ["cdylib", "lib"]
+
+[dependencies]
+solana-program = "1.18.16"
+
+[dev-dependencies]
+solana-program-test = "1.18.16"
+solana-sdk = "1.18.16"
+tokio = { version = "1.36.0", features = ["full"] }
```

No need for full features for the tests to run
```suggestion
tokio = "1.36.0"
```